src/pages/documentation/profile/api-reference.md

```diff
@@ -0,0 +1,167 @@
+import Tabs from './tabs/api-reference.md'
+
+# API reference
+
+## extensionVersion
+
+The `extensionVersion()` API returns the version of the Profile extension.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="6"/>
+
+iOS (ACP 2.x)
+
+<Tabs query="platform=ios-acp&api=extension-version"/>
+
+React Native
+
+<Tabs query="platform=react-native&api=extension-version"/>
+
+Flutter
+
+<Tabs query="platform=flutter&api=extension-version"/>
+
+Cordova
+
+<Tabs query="platform=cordova&api=extension-version"/>
+
+Unity
+
+<Tabs query="platform=unity&api=extension-version"/>
+
+Xamarin
+
+<Tabs query="platform=xamarin&api=extension-version"/>
+
+## getUserAttributes
+
+The `getUserAttributes()` API gets the user profile attributes with the given keys.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
+
+iOS (ACP 2.x)
+
+<Tabs query="platform=ios-acp&api=get-user-attributes"/>
+
+Flutter
+
+<Tabs query="platform=flutter&api=get-user-attributes"/>
+
+Cordova
+
+<Tabs query="platform=cordova&api=get-user-attributes"/>
+
+Xamarin
+
+<Tabs query="platform=xamarin&api=get-user-attributes"/>
+
+## registerExtension
+
+Registers the Profile extension with the Mobile Core extension.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="1"/>
+
+iOS (ACP 2.x)
+
+<Tabs query="platform=ios-acp&api=register-extension"/>
+
+## removeUserAttribute
+
+Removes the user profile attribute for the given key.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
+
+iOS (ACP 2.x)
+
+<Tabs query="platform=ios-acp&api=remove-user-attribute"/>
+
```

Comment:
missing react native samples - https://github.com/adobe/react-native-acpuserprofile#user-profile

Reply:
Updates made in https://github.com/AdobeDocs/acp-mobile-sdkdocs/pull/7/commits/95e8a39b0590c8b1da359b1707e2b83a3408414b